### PR TITLE
DM-43971: Update credentials

### PR DESCRIPTION
**Pull request by bsmartradio**
Update sample consumer code to reflect USDF endpoints

### CODE REVIEW

File: examples/alert_stream_integration_endpoint/aiokafka_consumer/aiokafka_consumer.py [hunk 1]
```diff
@@ -62,24 +59,19 @@ async def main():
         # These next two properties tell the Kafka client about the specific
         # authentication and authorization protocols that should be used when
         # connecting.
-        security_protocol="SASL_SSL",
+        ssl_context=None,
+        security_protocol="SASL_PLAINTEXT",
```
Comment (ebellm) [inline examples/alert_stream_integration_endpoint/aiokafka_consumer/aiokafka_consumer.py:63]: why did we switch to plaintext?

File: examples/alert_stream_integration_endpoint/java_console_consumer/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ The Java consumer relies upon a configuration file which holds the username and
 Make a file called 'consumer.properties' and fill it with this:
 
 ```
-security.protocol=SASL_SSL
+security.protocol=PLAINTEXT
```
Comment (ebellm) [inline examples/alert_stream_integration_endpoint/java_console_consumer/README.md]: why is this PLAINTEXT when it was SASL_PLAINTEXT above?

**Review by ebellm — APPROVED** [review]
Approving this for the broker discussion; I'd like to discuss SASL_SSL vs SASL_PLAINTEXT though.

File: examples/alert_stream_integration_endpoint/java_console_consumer/README.md (hunk 2 above)
Comment (bsmartradio) [reply examples/alert_stream_integration_endpoint/java_console_consumer/README.md]: I miss-typed that one, it should still be SASL_PLAINTEXT

File: examples/alert_stream_integration_endpoint/aiokafka_consumer/aiokafka_consumer.py (hunk 1 above)
Comment (bsmartradio) [reply examples/alert_stream_integration_endpoint/aiokafka_consumer/aiokafka_consumer.py:63]: We do not have encryption set up on the kafka cluster. We can do it, but it was discussed that the data does not need to be decrypted so there isn't a need to put that layer of encryption on.

**Pull request merged**